hangman.py

```diff
@@ -23,7 +23,7 @@
 
 """
 
-stored_letters = list("java")  # converts string to list
+# stored_letters = list("java")  # converts string to list
 guessed_word = len(stored_letters) * ["_"]
 used_letters = list()
 
```

Comment:
count_balance is going down so just with this it becomes an infinite loop.

I know there is a break inside the loop but having both conditions is confusing as to which one matters.

Reply:
I changed the confusing condition in the while loop. the committed file is in my development branch